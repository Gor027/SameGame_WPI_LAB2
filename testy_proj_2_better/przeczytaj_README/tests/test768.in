o`
`o` o `` `o`  o o` `o`o`o `o o```o o``o ````ooo  o `oo`  o`  ` `ooooo`o 
oo` `` o      o` `` 
``o o`o `o` o` oo``o` ` oo 
 o  ooo` `  oo`` `oooo  o`o`o` `o `
`o ` ` ` `` ` ` o ` o  `  `` o``  ``  `  o `  o`
o `` ` oo `o  `o oo  `` `o o ``o``oo`` o``
 ```o`oo` o o  `o `  ```o`` o o  o o  o  ``` `o` o `  o `` o````o ``   
``o     `   o   ``o``ooo`  
oooo````o`   oooo o ` ```o o ooo   ` `
o````ooo`o  o` ooo` `o``o``oo  ``   oo`o  o` o oo ` o` `` o  `o
`o ` o
 o` ooo `o o` o` `o``  o o```   o   oo`o   ` oooo  o   o` oo oo o`  o``o``o oo` 
`  o
`o   o `o` ` o `o ``o` o` o` ` `oo `` `o `o 
`o `o`o o` `o  oo    ` `````o o `o o``oooo
 `     oo `  `   o o ooo
 o` ` o` ooo``  
 o o`o ``o`
o  `oooo ``` o``o ` o``o``ooo o`o`o`oo `  `o ` o ``` o `o`o 
