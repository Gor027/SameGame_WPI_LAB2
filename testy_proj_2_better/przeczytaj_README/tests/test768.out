 `
 o   `
oo   ` `
``  oo o `        `
oo  `` ``o     `  `    `o
`o``o```o`     oo o o` `o  o
````o`oo``o o  `o`` `oo`oo``
o`o`o`oo``` o`ooo`o o``o``o`   ``  o
`o`o`oo`oo```oo`o```o```oo`o  ``o  ```o
o````o`o`o`ooo`o``oo`o`````ooo```o o```o ` `
ooo`ooo```ooo``ooooo`ooo`o`oo``o`` oo`oo o`` o `
`oooooooo```oo``o``````oo```oooooo``oo`o ``` `o`   `    ` `  `
`o````o`ooo```o`ooo``````oo```o`oo```o`o``o` ooo o ` o ``oo  `` o
`o`o`oo`oo`oo`ooo```oooo````ooo``ooo```o``o` ooooo`oo``o``oo `o`ooooo`o
````oooo`````o``oo`oo``o``ooo`o`o`o`ooo`oo`oo`ooo```ooo`o`oo`oo`o```o``o``ooo`
